### ISSUE TITLE
Website URL problems (new URL)

### ISSUE THREAD

**Issue by tkzv** [alsaplayer/alsaplayer#17]
Is this the place to report troubles with alsaplayer.org website? It seems to have been taken over by some online casino advertisers. No malware, at least.

**Comment by domichel** [comment]
The url has changed a while ago. The new one is http://alsaplayer.sourceforge.net/

It is also the project site at https://sourceforge.net/projects/alsaplayer/

I will keep that bug open for peoples to see it.

**Comment by luzpaz** [comment]
Website needs updated info about github source repo. Most link point to sourceforge for git access etc... (these should be changed)

**Comment by domichel** [comment]
> Website needs updated info about github source repo. Most link point to sourceforge for git access etc... (these should be changed)

I done some updates on both sites. Can you please check if it is OK, and report it back. And if not, paste the url(s) here.

**Comment by luzpaz** [comment]
Yes, this is better. The one thing that I would suggest ATM is maybe put an entry at the top of the **News** section: 
![Image](https://github.com/user-attachments/assets/2a361291-1b51-4339-ab80-ee0a9931c159)
explaining clearly that development is occurring on Github.

An aside, on the sourceforge side, you could also consider disabling the ticket system over there.

**Comment by luzpaz** [comment]
There is also the Changelog link in the sidebar that could be updated to either:  
* https://github.com/alsaplayer/alsaplayer/blob/master/ChangeLog  
OR  
* https://sourceforge.net/p/alsaplayer/news/

**Comment by luzpaz** [comment]
Documentation link in sidebar points to https://alsaplayer.sourceforge.net/docs.php?style=styles/Trinine/  
Is that the best target or is there something more modern ?

**Comment by luzpaz** [comment]
It would be nice to just simplify the website. Perhaps you can open a ticket asking for volunteers to help with that ?

**Comment by domichel** [comment]
I changed the default style to Dum. It's a light turquoise-blue style and I poked its css to get bigger fonts. But I didn't got the font in the News section to scale as I want. It is a css issue and I didn't have more time for that if I want to advance my todo list.

I also updated the News section with a link on github at the top, and with the 0.99.82 release.

Before I change more files, I must do a files cleanup because on the website repo, almost all files have 2 versions, and I want to remove the non used ones. It was for testing.

After that, I will make the other changes you suggested and a repo on github for that website.

When done, I will open a ticket.

**Comment by domichel** [comment]
I changed the song of the week. It was a long week. 🥇

**Comment by domichel** [comment]
I am doing the files cleanup, and at the same time I cleanup the content, one file at a time.

For the **ChangeLog**, I put a title at the top with a link on master/ChangeLog on github, and changed the existing title to **Old ChangeLog**. I want to keep it because some time ago, I was contacted by a corporatised copyright hunter. I said him to contact Andy, and to keep it is one proof that AlsaPlayer is into the public domain from a very long time.

**Comment by domichel** [comment]
I done several changes, including to let the browser use its default font size, and use font sizes related to it into the css. That default size can be defined into the browser preferences.

That solved the issue with the News section.

Can you please try it and report back?

**Comment by luzpaz** [comment]
Yes, these are the improvements that were needed for modernization! Bravo!

**Comment by luzpaz** [comment]
Just a heads up: the historical [changelog](https://alsaplayer.sourceforge.net/changelog.php?style=styles/Dum/) and the current [changelog](https://github.com/alsaplayer/alsaplayer/blob/master/ChangeLog) on github are 2 different style changelogs. The former is a better style IMHO because it summarizes commits instead of listing every single one made. Any chance you'd consider the previous approach to changelog going forward ?

**Comment by domichel** [comment]
I make the current [changelog](https://github.com/alsaplayer/alsaplayer/blob/master/ChangeLog) by running `git log > ChangeLog` in the sources.  If I can find a git command that can replicate the old style, I can put it into a script into  the sources.

**Comment by domichel** [comment]
The code of the web site can look complicated, but its file structure is simple. Each file include some common files: a top header file, a bottom header file and a file with the menu. And its css is simple, maybe too simple because it is still a few glitches with the font sizes and the images placement, but nothing that is visually really relevant now that its font size reference is the one fixed by the browser preferences.

**Comment by domichel** [comment]
I just made a repo for the website: [AlsaPlayerWeb](https://github.com/alsaplayer/alsaplayer.sourceforge.net) with https://github.com/alsaplayer/alsaplayer.sourceforge.net/issues/1 that will take the relay of this bug.

Thanks again!
